packages/font-glyphs/src/letter/latin/upper-h.ptl

```diff
@@ -195,17 +195,17 @@ glyph-block Letter-Latin-Upper-H : begin
 		select-variant "cyrl/enghe.\(suffix)" (follow -- 'cyrl/enghe/ghePart')
 
 		create-glyph "Hwair.\(suffix)" : glyph-proc
-			local df : include : DivFrame para.diversityM 3
+			local df : include : DivFrame para.diversityMM 3
 			include : df.markSet.capital
 			include : HwairShape df CAP XH slabType
 
 		create-glyph "cyrl/NjeKomi.\(suffix)" : glyph-proc
-			local df : include : DivFrame para.diversityM 3
+			local df : include : DivFrame para.diversityMM 3
 			include : df.markSet.capital
 			include : HwairShape df CAP (CAP / 2) slabType
 
 		create-glyph "cyrl/njeKomi.\(suffix)" : glyph-proc
-			local df : include : DivFrame para.diversityM 3
+			local df : include : DivFrame para.diversityMM 3
```

Comment:
Maybe for lowercase Komi letters we could still use `diversityM`. They are slightly too wide and the ail is not a "full" letter.

Reply:
This change is optional though.